implementations/scala-mjs-validator/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM sbtscala/scala-sbt:eclipse-temurin-17.0.4_1.7.2_2.13.10 AS builder
+
+RUN git clone https://gitlab.lip6.fr/jsonschema/modernjsonschemavalidator /opt/mjs
```

Comment:
Is there a reason you're building this yourself here rather than letting sbt do it? I'll post a suggestion (in the build.sbt) just in case you haven't considered that.

Reply:
I actually had not considered that. Thank you for the suggestion!